### PR TITLE
Fix nbconvert handler

### LINKED ISSUE #2974: server error when downloading PDF via LaTeX
I have installed a clear version of Python 3.6 and ran the following `$ pip install notebook` to install jupyter on my Windows machine.

When I try to download my notebook as PDF via LaTeX I receive the following error:

errorjupyter

nbconvert was installed as a dependency with jupyter. Furthermore, I have MikTex installed for LaTeX.

Worth mentioning, I have tried this with different python versions and on my Mac as well. 

Hopefully someone could shed some light on this error.

---- 
    
Originally opened as jupyter/nbconvert#697 by @MustiDarsh, migration requested by @mpacer

### PR DESCRIPTION
**Pull request by mpacer**
closes #2974, 

Other small improvements while I was doing this:

Building the resources dict separately from invocation is more readable. 

`resources['metadata']['name']` now will use the notebook level metadata.title value as passed through to name (if present) and fall bad to stripping the extension using os.path.splitext. This notion of `name` is only used in the html and slides exporters to create a `<title/>` tag.

### CODE REVIEW

**Comment by mpacer** [comment]
I can't restart the Appveyor build… it looks like that error might just go away since i just verified that the conda-forge package does exist…

But for some reason I can't log in to appveyor,

**Comment by mpacer** [comment]
@gnestor Pdf exports were broken with 5.2 (#2974), so this should be prioritised in a patch release.

**Comment by mpacer** [comment]
Can someone please add me to the list of those with permissions to restart appveyor builds? I don't understand why it doesn't just use github permissions, but it seems to not do that.

**Comment by gnestor** [comment]
@Carreau Do you have those powers?

**Comment by mpacer** [comment]
@gnestor I really don't think this should wait until 5.3… instead aiming at 5.2.1 or are you using 5.3 to also organise patch releases for 5.2.x?

**Comment by Carreau** [comment]
> @Carreau Do you have those powers?

Not sure. I tried something, @mpacer you should receive an invite I guess from AppVeyor, if it does not work I think we need to as Kyle.

**Comment by gnestor** [comment]
We don't have a milestone for 5.2.1 so I marked it as 5.3. I'll create one now and shoot for releasing 5.2.1 tomorrow.

File: notebook/nbconvert/handlers.py [hunk 1]
```diff
@@ -76,40 +76,50 @@ def get_exporter(format, **kwargs):
...
             # not a notebook, redirect to files
             return FilesRedirectHandler.redirect_to_files(self, path)
 
+        nb = model['content']
+
         self.set_header('Last-Modified', model['last_modified'])
 
+        # create resources dictionary
+        mod_date = model['last_modified'].strftime(text.date_format)
+        nb_title = nb.metadata.get("title","") or os.path.splitext(name)[0]
```
Comment (takluyver) [inline notebook/nbconvert/handlers.py]: Not convinced about the title from notebook metadata taking precedence over the filename used to identify it. The notebook metadata is already available in the template, it doesn't need to be added to resources.

**Comment by gnestor** [comment]
The 5.2.1 branch is ready for release so we're just waiting on this PR 👍

File: notebook/nbconvert/handlers.py (hunk 1 above)
Comment (mpacer) [reply notebook/nbconvert/handlers.py]: Fine, this was a fix that I had added on the nbconvert side for the LaTeX interpreter and was just trying to keep it consistent here. But I'm not wedded to it.

**Comment by mpacer** [comment]
fixed!

**Comment by gnestor** [comment]
I tested and export to PDF is working 👍 

@mpacer Shall I merge?

**Comment by minrk** [comment]
@gnestor 🚢

**Comment by gnestor** [comment]
@meeseeksdev backport to 5.2.1

**Comment by gnestor** [comment]
I backported this manually to 5.2.1 👍

**Comment by mpacer** [comment]
Apologies! didn't see you ask me if it could be merged — I had intended it to be done, yes. Any thoughts as to why the meseeksdev backport didn't work?

**Comment by gnestor** [comment]
No prob! No I don't have access to the meseeksdev logs but there must've been a merge conflict

**Pull request merged**